### ISSUE TITLE
 [Question/Feature] 

### ISSUE THREAD

**Issue by moncat2005** [trickerer/Trinity-Bots#511]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that my issue may get **closed without notice** if I *intentionally* remove or skip any mandatory\* field
- [X] This is NOT a bug/crash report
- [X] I've searched the bugtracker for similar questions/requests **including closed ones**

### Description

Sorry, my previous question may not have been described clearly.
 I am not referring to NPC_ TEXT and NPC_TEXT_ locale application issues. 
 As shown in the figure, how to convert COMMAND LINE (Looking for bots of class[class id to display text]) into zhCN language.
like human,orc,dwarf convert 人类，兽人，矮人。

![class](https://github.com/trickerer/Trinity-Bots/assets/34859687/ca51a8a2-9f0c-434e-bcb8-439f28a910e2)
Thanks!

### TrinityCore or AzerothCore

AzerothCore

**Comment by lemonmomomo** [comment]
@moncat2005 其实你说的这个问题可以在creature_template_locale中自行添加修改。

**Comment by trickerer** [comment]
Commands don't get localized because as administrator you are expected to know English. I'm not speaking about NPCBots commands in particular, this is just a concept and applies to any command.

The only commands that may need a translation is the commands that are available to players (GM level 0). But even existing command string are by default barely transated in AC and not translated at all in TC.
Making a command viable for localization required involving `trinity_string` / `acore_string` tables into NPCBots mod. Which I am not in a hurry to do.

**Comment by moncat2005** [comment]
Perhaps there are some issues with my application method.
I did not use NPC70000 to make players call npcbots.
Based on providing players with a friendly NPCbots interface, I recommend players to using Addons call npcbots. (NetherBot for language conversion). 
When players use NetherBot to select and view the npcbots list, I will creature_template_npcbot_appearance name field of appearance has been changed to Chinese, but I am unable to convert the NPC's race/class information into a language, which is somewhat regrettable.
